.gitignore

```diff
@@ -35,3 +35,128 @@ jspm_packages
 
 # Optional REPL history
 .node_repl_history
+
+
+.db
```

Comment:
You didn't gitignore your db folder correctly, so it still got pushed up. Take off the dot and just put ``db``